### PR TITLE
Bounty List: Fix loaded state when using filters of languages

### PR DESCRIPTION
**Pull request by ChetanKarwa**
## Issue ticket number and link
https://github.com/stakwork/sphinx-tribes-frontend/issues/38

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by ChetanKarwa** [comment]
image

Fixed No Results state

**Comment by ecurrencyhodler** [comment]
Hey Chetan, you have a prettier test failing. Can you address that? Also this pr should include a test as defined in the acceptance criteria. Thanks!

File: src/people/utils/NoResults.tsx [hunk 1]
```diff
@@ -25,11 +25,11 @@ const H = styled.div`
   letter-spacing: 0px;
   color: rgb(60, 63, 65);
 `;
-function NoResults() {
+function NoResults({loaded = false}: {loaded?: boolean}) {
   const { ui } = useStores();
   const { searchText } = ui || {};
-
-  if (searchText) {
+  console.log({searchText})
```
Comment (elraphty) [inline src/people/utils/NoResults.tsx]: Remove the console.log() statement

**Comment by ecurrencyhodler** [comment]
Tested and it works. Good job! Paid.

**Pull request merged**